kanalytics/src/commonMain/kotlin/com/addhen/kanalytics/KAnalyticsEvent.kt

```diff
@@ -14,14 +14,17 @@ public class KAnalyticsEvent(
   public val eventName: String,
   public val eventDescription: String? = null,
   properties: Map<String, Any?> = emptyMap(),
+  userProperties: Map<String, Any?> = emptyMap(),
```

Comment:
Can this be data class?

Reply:
Yes it can be but I thought it should just be a field as `KAnalyticsEvents` itself is a a type consumed by the tracker implementations so it being a field should be good enough.